torch/testing/_internal/common_methods_invocations.py

```diff
@@ -64,6 +64,7 @@ def __init__(self,
                  dtypesIfCPU=None,  # dtypes this function is expected to work with on CPU
                  dtypesIfCUDA=None,  # dtypes this function is expected to work with on CUDA
                  dtypesIfROCM=None,  # dtypes this function is expected to work with on ROCM
+                 tested_dtypes=None,  # dtypes to test with by default
```

Comment:
How about "tested_dtypes" -> "default_test_dtypes"?

And the comment can elaborate that this iterable is intersected with the device-specific dtype iterables above.